### DISCUSSION TITLE
可否做一个您平时用的软件分享

### DISCUSSION THREAD

**Discussion by zhaoliang0302** [ShixiangWang/self-study#79] [Q&A]
翔哥好，一直觉得您的软件和软件配置都很有品位，基本都是最佳方案，可否做一期相关的分享呢？比如最近在用zotero发现没有那么好用，您应该有比较好的配置方案😁

**Comment by ShixiangWang** [comment]
软件用的多才比较有心得吧～我其实还好。我自己文献管理用的就是 zotero，就很简单的使用，没有什么技巧。 你可以列一下你想了解的，我看看那些我比较有经验，博士毕业答辩完成后我抽空分享下。
